### PR TITLE
Update get and identify APIs to return the item/id in one item case.

### PR DESCRIPTION
**Pull request by lzhoucs**
**Type:** feature

**Description:** 
Update get and identify APIs to return the item/id instead of the one item.id array in one item case.

**Related Issue:** #60 

* [x] There is a related issue
* [x] All contributors have signed a CLA
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [ ] The code passes the CI tests
* [x] Unit or Functional tests are included in the PR
* [x] The PR increases or maintains the overall unit test coverage percentage
* [x] The code is ready to be merged

### CODE REVIEW

File: src/store/createStore.ts [hunk 1]
```diff
@@ -83,12 +86,16 @@ function isPatch(patchObj: any): patchObj is {id: string; patch: Patch<any, any>
 }
 
 const createStore: StoreFactory = compose<Store<{}, {}, any>, StoreOptions<{}, {}>>({
-	get(this: Store<{}, {}, any>, ids: string[] | string): Promise<{}[]> {
+	get(this: Store<{}, {}, any>, ids: string[] | string): Promise<any> {
```
Comment (maier49) [inline src/store/createStore.ts]: Could we type the return here to `Promise<{} | {}[]>`? That way it provides at least some guarantee about the return type in the implementation.

File: src/store/createStore.ts [hunk 2]
```diff
@@ -222,8 +229,14 @@ const createStore: StoreFactory = compose<Store<{}, {}, any>, StoreOptions<{}, {
 		});
 	},
 
-	identify(this: Store<{}, {}, any>, items: {}[] | {}) {
-		return instanceStateMap.get(this).storage.identify(Array.isArray(items) ? items : [ items ]);
+	identify(this: Store<{}, {}, any>, items: {}[] | {}): any {
```
Comment (maier49) [inline src/store/createStore.ts:233]: Same question here really, could this return `string | string[]`?

**Review by maier49 — CHANGES_REQUESTED** [review]

File: src/store/createStore.ts (hunk 1 above)
Comment (lzhoucs) [reply src/store/createStore.ts]: Good point. I have made the change.

File: src/store/createStore.ts (hunk 2 above)
Comment (lzhoucs) [reply src/store/createStore.ts:233]: Although `get` can be done this way, `identify` can't as I tried. Please let me know if you have ways to get around this and I am happy to make it type safer.

**Comment by lzhoucs** [comment]
Just a note, the CI has been hanging on `iphone 9.3` which is not caused by changes in this PR:
```
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
```
I've seem this happen before. It seems it only fails occasionally.

**Review by maier49 — APPROVED** [review]

**Pull request merged**